CHANGELOG.md

```diff
@@ -12,6 +12,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 - impl `BorshSchema` for `Rc` and `Arc`; add doc for `rc` feature ([#268](https://github.com/near/borsh-rs/pull/268))
 
+### Other
```

Comment:
please, revert change to `CHANGELOG.md`, it makes `CHANGELOG.md` look like 1.3.0 was modified retroactively. changelog entry will be computed by `releaze-plz`  for next release